project/Http4sPlugin.scala

```diff
@@ -34,9 +34,9 @@ object Http4sPlugin extends AutoPlugin {
   override lazy val buildSettings = Seq(
     // Many steps only run on one build. We distinguish the primary build from
     // secondary builds by the Travis build number.
-    http4sApiVersion := version.map {
-      case VersionNumber(Seq(major, minor, _*), _, _) => (major.toInt, minor.toInt)
-    }.value,
+    http4sApiVersion := version.map { case VersionNumber(Seq(major, minor, _*), _, _) =>
+      (major.toInt, minor.toInt)
+    }.value
```

Comment:
Hmm wasn't me that changed this, I did run `scalafmtCheck` though. Should probably revert these changes?

Reply:
Ah, I think I got `scalafmt` to run automatically and it probably was never run for this file, hence it was formatted when I touched it.

Reply:
We haven't been running scalafmt on sbt.  I'd like to, but we've been deferring that until the branch situation settles.

If you have a moment to revert that to keep this diff smaller, that would great.  If not, I can sort it out.

Reply:
I have reverted the changes and force pushed to keep clean history, hope that's ok 😬